plaidml/edsl/__init__.py

```diff
@@ -600,6 +600,27 @@ def TensorIndexes(count):
     return [TensorIndex() for i in range(count)]
 
 
+def Placeholder(dtype_or_shape, dims=[], name=''):
+    """Creates a placeholder tensor, specified either by a combination of data
+       type and dimensions or by passing in a LogicalShape.
+
+        Args:
+            dtype_or_shape (DType | LogicalShape): Specifies either the data 
+            type or the logical shape of the ``Placeholder``.
+            dims (list): Specifies the dimensions of the ``Placeholder``.
+            name (string): Optional name to be assigned to the ``Tensor``.
```

Comment:
These are still considered kwargs, but I'm not sure if napolean cares about that or not?

Reply:
It doesn't care about keyword args like this necessarily, but it has a separate way of letting you denote what goes into `*args` and `**kwargs`